.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
Can you go into a bit of detail about why this file is here and what it does please

Reply:
These are output from the Extension Generator.

I might be wrong , but I left them because I understood as follows;

- typescript files (src/\*.ts) are converted into javascript files (out/\*.js).
- files.exclude: VS code extension uses \*.js (not \*.ts)
- search.exclude: developer need to search source code (\*.ts), but not intermediate files (\*.js).
- typescript.tsc.autoDetect: settings in package.json make the feature unnecessary